client/src/form/AddNewListBar.js

```diff
@@ -50,48 +62,49 @@ export default function BasicTextFields(props) {
             <Grid item xs={10} md={8} style={{textAlign:'center'}}>
-                <TextField 
-                    fullWidth
-                    label="Title"
+            <Typography component="h5" style={{fontWeight: 'bold'}}>
+                Add a title *
+            </Typography>
+            <Box className={classes.box} borderRadius={50} flexGrow={1}>
+                <InputBase 
                     id="list-title" 
-                    color="secondary"
                     name="title" 
-                    placeholder="Enter list title" 
-                    variant="standard" 
+                    placeholder="Enter list title"  
                     onChange={(e) => onChange(e)}
                     className={classes.text}
                     inputProps={{ maxLength: 20 }}
-                    helperText="max length 20 charaters"
                     required
                     />
-                <TextField 
-                    fullWidth
-                    label="Image Url"
+            </Box>
+            <Typography component="h5" style={{fontWeight: 'bold'}} >
+                Add a cover
+            </Typography>
+            <Box className={classes.box} borderRadius={50} flexGrow={1}>
+                <InputBase 
                     id="list-image" 
-                    color="secondary"
                     name="imageurl" 
                     placeholder="Paste list cover image url" 
-                    variant="standard"
                     onChange={(e) => onChange(e)} 
                     className={classes.text}
                     required
                     />
-                <TextField 
-                    fullWidth
-                    multiline
-                    label="List Description"
-                    name="listdescription"
-                    id="list-description" 
-                    color="secondary" 
-                    placeholder="My shopping list" 
-                    variant="standard" 
-                    onChange={(e) => onChange(e)}
-                    className={classes.text}
-                    inputProps={{ maxLength: 200 }}
-                    helperText="max length 200 charaters. Default: My shopping list"
-                    />
+                </Box>
+                <Typography component="h5" style={{fontWeight: 'bold'}} >
+                    Add a description
+                </Typography>
+                <Box className={classes.box} borderRadius={50} flexGrow={1}>
+                    <InputBase 
+                        multiline
+                        name="listdescription"
+                        id="list-description" 
+                        placeholder="My shopping list" 
+                        onChange={(e) => onChange(e)}
+                        className={classes.text}
+                        inputProps={{ maxLength: 200 }}
+                        />
```

Comment:
`inputProps={{ maxLength: 200,style: { textAlign: 'center' } }}` may help us to center the placeholder and input text.